src/res/SfmlFileResource.hpp

```diff
@@ -1,5 +1,5 @@
-#ifndef ChessPlusPlus_Res_SfmlFileResourceClass_HeaderPlusPlus
-#define ChessPlusPlus_Res_SfmlFileResourceClass_HeaderPlusPlus
+#ifndef chesspp_res_SfmlFileResource_HeaderPlusPlus
+#define chesspp_res_SfmlFileResource_HeaderPlusPlus
 
 #include "ResourceManager.hpp"
 //#include "SFML.hpp"
```

Comment:
I know it's part of this PR, but we should definitely remove commented out code (includes)